src/zeit/migrate/migrate.py

```diff
@@ -0,0 +1,159 @@
+import UserDict
+import argparse
+import contextlib
+import lxml.etree
+import lxml.objectify
+import re
+import tinydav
+import urlparse
+
+NAMESPACE = 'http://namespaces.zeit.de/CMS/document'
```

Comment:
Nutze jetzt den selben Namen wie in `zeit.cms`, also `DOCUMENT_SCHEMA_NS`. Ich denke das reicht vorerst (anderen Namespace brauchen wir nicht und als separates Paket haben wir keinen Zugriff auf `zeit.cms`).